### PR TITLE
Added talks to call for participation

### PR DESCRIPTION
**Pull request by jdamerow**
## Description
Added a talk section to the participate page.

## Motivation and Context

To encourage submissions by people who are not interested in writing a paper.

## Checklist:

- [x] I have posted the link for the PR in the usrse slack (#usrse-2023-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [x] I have updated the README.md if necessary

cc @mrmundt @cabejackson @manning-ncsa @jmelot @jbteves @jscubida

### CODE REVIEW

File: pages/participate.md [hunk 1]
```diff
@@ -57,7 +57,16 @@ Papers should follow the [IEEE format](https://www.ieee.org/conferences/publishi
 - **Camera-ready papers / notebooks due**: Monday, June 26, 2023
 - **Registration for authors of papers/notebooks**: Monday, June 26, 2023
 
+## Talks
 
+We invite talks highlighting the above topic areas and the software/software techniques that enabled the research. Accepted talks will be given the opportunity to have a 15 minutes oral presentation during the conference. Talk submissions should be in the form of an abstract, no more than 1 page in length.
```
Comment (mrmundt) [inline pages/participate.md:62]:
```suggestion
We invite talks highlighting the above topic areas and the software/software techniques that enabled the research.
Accepted talks will be given 15 minutes for an oral presentation during the conference.
Talk submissions should be in the form of an abstract, no more than 1 page in length.
```

Comment (mrmundt) [reply pages/participate.md:62]: I would prefer we do a word limit over a page limit here. We don't define a standard format, font size, etc., so "one page" seems too vague. Can we say 500 words instead? (Or whatever value?)

File: pages/participate.md [hunk 2]
```diff
@@ -57,7 +57,16 @@ Papers should follow the [IEEE format](https://www.ieee.org/conferences/publishi
...
+
+**Timeline**
+
+- **Submissions open**: Tuesday, February 16, 2023
```
Comment (mrmundt) [inline pages/participate.md:66]:
```suggestion
- **Submissions open**: Thursday, February 16, 2023
```

**Review by mrmundt — CHANGES_REQUESTED** [review]

File: pages/participate.md (hunk 1 above)
Comment (jdamerow) [reply pages/participate.md:62]: I do too, but that's what we have said for posters, notebooks, tutorials as well. I'm up for changing it across all categories, not sure though if that's a good idea after we have already published it?

Comment (mrmundt) [reply pages/participate.md:62]: Ehh... We already have it in other places, so okay, let's keep it here, too. But worth noting for next year!

**Review by mrmundt — APPROVED** [review]

**Pull request merged**